deploy/upgrade-create-new-version.sh

```diff
@@ -0,0 +1,39 @@
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2019 Red Hat, Inc.
+#
+
+set -ex
+
+LATEST_VERSION=$(ls -d /registry/kubevirt-hyperconverged/*/ | sort -r | head -1 | cut -d '/' -f 4); 
+UPGRADE_VERSION=100.0.0
+
+cp -rf /registry/kubevirt-hyperconverged/$LATEST_VERSION /registry/kubevirt-hyperconverged/$UPGRADE_VERSION
+
+mv /registry/kubevirt-hyperconverged/$UPGRADE_VERSION/kubevirt-hyperconverged-operator.v$LATEST_VERSION.clusterserviceversion.yaml /registry/kubevirt-hyperconverged/$UPGRADE_VERSION/kubevirt-hyperconverged-operator.v$UPGRADE_VERSION.clusterserviceversion.yaml
```

Comment:
Let me take a look to see if I can use hack/build-manifest.sh.

Reply:
As long as the Dockerfile is in /deploy, it will not work. The docker build doesn't have access to files outside of /deploy. @rthallisey do you foresee any issue with moving registry image Dockerfiles to the repository root?

Reply:
Would [deploy/Dockerfile](https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/deploy/Dockerfile) need to be moved or the new ones you're creating?

Reply:
> do you foresee any issue with moving registry image Dockerfiles to the repository root?

the only issue that stands out is not being able to `operator-sdk build ...` in this project. It's probably worth avoiding deviations away from operator-sdk patterns unless we have a really good reason.

There are some manipulations of the path you give to the Docker daemon that may do what you want but it isn't clear what we are trying to do.

Reply:
@rthallisey, both would need to be moved.

@djzager, this PR has a new script that creates a new CSV version that is used during the upgrade test. Ryan pointed out that hack/build-manifest could be used instead of the new script to generate the new CSV version. The problem with using hack/build-manifest.sh is that it is outside of the docker build path (/deploy). You mentioned some manipulations of the path that would be done. Can you point me to some examples? Thanks.

An alternative would be to move hack/build-manifest.sh, tools/manifest-templator, and any other files it needs to /deploy. Would this cause issues?

Reply:
We can move `hack/build-manifests.sh` & `tools/manifest-templator`, then create sym links from their original paths.  That should preserve the current workflow and give you access to those file.